src/test/scala/nl/hugo/redbook/ch7/ExecutorServiceDecorator.scala

```diff
@@ -0,0 +1,11 @@
+package nl.hugo.redbook.ch7
+
+import java.util.concurrent.{ ExecutorService, ThreadPoolExecutor }
+
+object ExecutorServiceDecorator {
+
+  implicit class ExectorServiceOps(es: ExecutorService) {
```

Comment:
Actually I am using it in many tests, because the ExecutorService doesn't expose  getCompletedTaskCount from the ThreadPoolExecutor.